lib/tasks/evm_dba.rake

```diff
@@ -104,6 +104,26 @@ namespace :evm do
       Rake::Task['db:seed'].invoke
     end
 
+    GC_DEFAULTS = {
```

Comment:
is it possible to reformat these as the `:name => nil` type arguments that are used by `vacuum_args`?

Reply:
Going to pass on this change for the first pass, as I would like this to be close to a 1-to-1 conversion from `PostgresAdmin`.  Leave the refactor PR to a follow up.